Formula/inadyn.rb

```diff
@@ -0,0 +1,41 @@
+class Inadyn < Formula
+  desc "Dynamic DNS client with IPv4, IPv6, and SSL/TLS support"
+  homepage "http://troglobit.com/projects/inadyn/"
+  url "https://github.com/troglobit/inadyn/releases/download/v2.5/inadyn-2.5.tar.gz"
+  sha256 "f7faf0be4f0b4bfa1acc811189a9ed0a58bc367e48ea31c283920a2ef27cdc40"
+
+  patch do
+    url "https://github.com/troglobit/inadyn/commit/57bdcc0321b49ee68397c70140d9895655edb06f.patch"
```

Comment:
Please use `https://github.com/troglobit/inadyn/commit/57bdcc03.diff?full_index=1` as URL (and adapt the checksum below). It is more stable over time.

Reply:
`  * C: 12: col 3: `depends_on` (line 12) should be put before `patch` (line 7)`